deploy/helm/charts/values.yaml

```diff
@@ -180,3 +180,28 @@ crds:
     volumeSnapshots:
       # Install Volume Snapshot CRDs
       enabled: true
+
+# zfsExporter Confiration
+zfsExporter:
+  enable: true
+  serviceMonitor:
+    enabled: false
+    path: /metrics
+    interval: 10s
+  name: zfs-exporter
+  port: 9134
+  image:
+    # Make sure that registry name end with a '/'.
+    # For example : registry.k8s.io/ is a correct value here and quay.io is incorrect
+    registry: harbor.obmondo.com/
+    repository: obmondo/zfs_exporter 
```

Comment:
Hi @Deveshs23 Many thanks for your contribution. We would like to understand that where is the code for the image that is being referred here lives? How does it operate with openebs zfs-localpv?
We would appreciate if the code lives in this repo, that the community will be able to maintain and extend it.



Also the monitoring stack for zfs is already present. Please check [openebs/monitoring ](https://github.com/openebs/monitoring) repo.



We currently export these [metrics](https://github.com/openebs/zfs-localpv/blob/develop/docs/prometheus-monitoring.md#zfs-metrics-exposed-by-prometheus) for zfs.



You are welcome to add to the above list of metrics. In that case please follow this:
1. Add the metrics/exporter code to [zfs-localpv](https://github.com/openebs/zfs-localpv) repo.
2. Add the monitoring code to [monitoring](https://github.com/openebs/monitoring) repo.
3. Add the helm chart changes if needed to [monitoring helm charts](https://github.com/openebs/monitoring/tree/develop/deploy/charts).